packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -1,3 +1,5 @@
+import 'server-only';
```

Comment:
I've import it separately on `auth` and `currentUser` only instead of importing it once on `server` index. 

My concern is that the error would be thrown even when importing type definitions, but I'd like to minimize the surface here.